aten/src/ATen/nn_parse.py

```diff
@@ -295,10 +295,10 @@ def initialize_output_arg(arg):
         arg['is_nullable'] = True
 
         # grad_weight and grad_bias need to be resized and zeroed
-        if arg['name'] == 'grad_weight':
+        if arg['name'] == 'grad_weight' and base['name'] != '_thnn_conv2d':
```

Comment:
If there's gonna be more of these, maybe it's worth like `PORTED_RESIZE_AND_ZERO = [...]` that gets checked against? 

Actually maybe worth it already given the multiple checks

Reply:
there are only two of them, so I didn't think it was worth it.